element/build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+plugins {
+    kotlin("multiplatform")
+    id("org.jmailen.kotlinter")
+    jacoco
+    id("org.jetbrains.dokka")
+    id("com.vanniktech.maven.publish")
+}
+
+apply(from = rootProject.file("gradle/jacoco.gradle.kts"))
+
+jacoco {
+    toolVersion = "0.8.7"
+}
+
+kotlin {
+    explicitApi()
+
+    jvm()
+    js().browser()
+
+    sourceSets {
+        all {
+            languageSettings.enableLanguageFeature("InlineClasses")
+        }
```

Comment:
Is the opt-in needed for Kotlin 1.6.0?

Reply:
Resolving this, handling in the other thread.